remote-query/remote-query-server/src/test/java/org/infinispan/query/remote/impl/ProtobufFieldIndexingMetadataTest.java

```diff
@@ -43,10 +43,8 @@
 
    protected EmbeddedCacheManager createCacheManager() throws Exception {
       ConfigurationBuilder cfg = getDefaultStandaloneCacheConfig(true);
-      cfg.transaction().transactionMode(TransactionMode.TRANSACTIONAL)
-            .indexing().enable()
-            .addProperty("directory.type", "local-heap");
-      return TestCacheManagerFactory.createCacheManager(cfg);
+      cfg.transaction().transactionMode(TransactionMode.TRANSACTIONAL);
+      return TestCacheManagerFactory.createServerModeCacheManager(cfg);
```

Comment:
+1 for createServerModeCacheManager instead of  createCacheManager but -1 for the removal of ```.indexing().enable()```

That removal just papers over another aspect: we should really add an SCI here that registers the schema from ```PROTO_DEFINITIONS``` instead of registering it manually in testProtobufFieldIndexingMetadata